services/src/handlers/ebv.rs

```diff
@@ -331,6 +331,28 @@ impl<C: SessionContext> Task<C::TaskContext> for EvbMultiOverviewTask<C> {
         EBV_MULTI_OVERVIEW_TASK_TYPE
     }
 
+    fn task_description(&self) -> String {
+        let files = tokio::runtime::Handle::current().block_on(async {
+            with_netcdfcf_provider(self.ctx.as_ref(), move |provider| {
+                provider.list_files().map_err(|_| {
+                    NetCdfCf4DProviderError::CdfCfProviderCannotListFiles {
+                        id: NETCDF_CF_PROVIDER_ID,
+                    }
+                })
+            })
+            .await
+        });
+        let files = match files {
+            Ok(x) => x,
+            Err(_) => return "Failed retrieve file list".to_string(),
```

Comment:
isn't this more something for the task status and not the description? Maybe you have to store the target file (or whether ALL overviews should be built) during the task creation and just access them here? But maybe there is another way of getting this info here.

Reply:
[d052c3b](https://github.com/geo-engine/geoengine/pull/820/commits/d052c3b88574c72dd7eda86fd6063ad8d0f0dbc0)